includes/specials/SpecialDeletedContributions.php

```diff
@@ -344,19 +365,19 @@ function getSubTitle( $userObj ) {
 		} else {
 			$user = Linker::link( $userObj->getUserPage(), htmlspecialchars( $userObj->getName() ) );
 		}
-		$nt = $userObj->getUserPage();
-		$id = $userObj->getID();
-		$talk = $nt->getTalkPage();
+		$userTitle = $userObj->getUserPage();
+		$userId = $userObj->getID();
+		$talk = $userTitle->getTalkPage();
 		if( $talk ) {
 			# Talk page link
 			$tools[] = Linker::link( $talk, $this->msg( 'sp-contributions-talk' )->escaped() );
-			if( ( $id !== null ) || ( $id === null && IP::isIPAddress( $nt->getText() ) ) ) {
+			if( ( $userId !== null ) || ( $userId === null && IP::isIPAddress( $userTitle->getText() ) ) ) {
 				if( $this->getUser()->isAllowed( 'block' ) ) { # Block / Change block / Unblock links
 					/* Wikia change begin - SUS-92 */
 					if ( $userObj->isBlocked( true, false ) && $userObj->getBlock( true, false )->getType() !== Block::TYPE_AUTO ) {
 					/* Wikia change end */
 						$tools[] = Linker::linkKnown( # Change block link
-							SpecialPage::getTitleFor( 'Block', $nt->getDBkey() ),
+							SpecialPage::getTitleFor( 'Block', $userTitle->getDBkey() ),
 							$this->msg( 'change-blocklink' )->escaped()
 						);
 						$tools[] = Linker::linkKnown( # Unblock link
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)